Update defender-for-storage-introduction.md (#104446)
Azure/azure-content: articles/defender-for-cloud/defender-for-storage-introduction.md

You can learn more by watching this video from the Defender for Cloud in the Field video series:
- [Defender for Storage in the field](episode-thirteen.md)

> > [!NOTE]

Reviewer: Update this to match the note in the storage docs.

> Microsoft Defender for Storage customers can now choose to move to a new predictable pricing plan. The pricing model is per-storage account, where high-volume transactions may incur additional overage charges. This new pricing plan will also include all new security features and detections.
>
> Customers using the legacy per-transaction pricing plan need to migrate to the new per-storage account plan to access these new features and pricing. The legacy per-transaction pricing plan charges are based on the number of analyzed transactions in the storage account.
>

> > For further details, please refer to the [Microsoft Defender for Storage FAQ](#faq---microsoft-defender-for-storage-pricing).

Reviewer: ```suggestion
> For further details, please refer to the [Microsoft Defender for Storage FAQ](azure-defender-storage-configure.md#faq---microsoft-defender-for-storage-pricing).
```

> > For further details, please refer to the [Microsoft Defender for Storage FAQ](azure-defender-storage-configure.md#faq---microsoft-defender-for-storage-pricing).

Reviewer: ```suggestion
> For further details, please refer to the [Microsoft Defender for Storage FAQ](../storage/common/azure-defender-storage-configure.md#faq---microsoft-defender-for-storage-pricing).
```

> > Microsoft Defender for Storage customers can now choose to move to a new predictable pricing plan. The pricing model is per-storage account, and high-volume transactions may incur additional overage charges. This new pricing plan will also include all new security features and detections.
> >
> > Customers using the legacy per-transaction pricing plan will need to proactively migrate to the new per-storage account plan to access these new features and pricing. The legacy per-transaction pricing plan charges based on the number of analyzed transactions in the storage account.
> >
> > For further details, please refer to the [Microsoft Defender for Storage FAQ](https://learn.microsoft.com/azure/storage/common/azure-defender-storage-configure?toc=%2Fazure%2Fdefender-for-cloud%2Ftoc.json&tabs=per-storage-account#faq---microsoft-defender-for-storage-pricing).

Reviewer: ```suggestion
> Microsoft Defender for Storage customers can now choose to move to a new predictable pricing plan. The pricing model is per-storage account, where high-volume transactions may incur additional overage charges. This new pricing plan will also include all new security features and detections.
>
> Customers using the legacy per-transaction pricing plan need to migrate to the new per-storage account plan to access these new features and pricing. The legacy per-transaction pricing plan charges are based on the number of analyzed transactions in the storage account.
>
> For further details, please refer to the [Microsoft Defender for Storage FAQ](#faq---microsoft-defender-for-storage-pricing).
```

### Can I exclude a specific Azure Storage account from a protected subscription? 

> Excluding specific storage accounts from protection is only possible on the per-transaction pricing plan. To exclude a storage account, follow instructions in [Exclude a storage account from a protected subscription in the per-transaction plan](defender-for-storage-exclude.md). 

Reviewer: ```suggestion
Excluding specific storage accounts from protection is only possible on the per-transaction pricing plan. The per-storage account pricing plan will support exclusion in the future. To exclude a storage account, follow instructions in [Exclude a storage account from a protected subscription in the per-transaction plan](defender-for-storage-exclude.md).
```

> Excluding specific storage accounts from protection is only possible on the per-transaction pricing plan. The per-storage account pricing plan will support exclusion in the future. To exclude a storage account, follow instructions in [Exclude a storage account from a protected subscription in the per-transaction plan](defender-for-storage-exclude.md). 

Reviewer: ```suggestion
Excluding specific storage accounts from protection is only possible on the per-transaction pricing plan. To exclude a storage account, follow instructions in [Exclude a storage account from a protected subscription in the per-transaction plan](defender-for-storage-exclude.md). 
```
